release_notes/topics/15_0_0.adoc

```diff
@@ -0,0 +1,9 @@
+= Highlights
+
+== Financial-grade API (FAPI) Improvements, FAPI CIBA and Open Banking Brasil
+
+The {project_name} server has improved support for the Financial-grade API (FAPI). More specifically, {project_name} is now compliant with FAPI CIBA and with OpenBanking Brasil.
+We also have support for CIBA ping mode. Thanks to https://github.com/tnorimat[Takashi Norimatsu], who did most of the work on FAPI CIBA and who is
+continuously doing really awesome job for the {project_name} project. Also thanks to https://github.com/DmitryMishchuk[Dmytro Mishchuk],
```

Comment:
should be "continually"
Continuously means non-stop, such an alarm that has to be turned off.
Continually means repeatedly, such as phone that keeps ringing.
Add "a" before "really."

Reply:
Thanks for explaining!